netebpfext/user/fwp_um.cpp

```diff
@@ -650,6 +653,10 @@ _IRQL_requires_max_(DISPATCH_LEVEL) NTSTATUS
 _IRQL_requires_max_(DISPATCH_LEVEL) NTSTATUS FwpsFlowAssociateContext0(
     _In_ uint64_t flow_id, _In_ UINT16 layer_id, _In_ uint32_t callout_id, _In_ uint64_t flowContext)
 {
+    if (ebpf_fault_injection_inject_fault()) {
+        return STATUS_NO_MEMORY;
```

Comment:
https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/fwpsk/nf-fwpsk-fwpsflowassociatecontext0 does not mention STATUS_NO_MEMORY in the list, but it does say "other return values".  Can this actually be returned by WFP?   Does this actually do a memory allocation in WFP?

Reply:
Yes, from the internal source code of this API, it does allocate
WFP_POOL_ALLOC_NONPAGED